programs/uxd/src/error.rs

```diff
@@ -74,7 +74,11 @@ pub enum UxdError {
     InvalidDepositoryCollateral,
     #[msg("The provided depository shares does not match the depository's one.")]
     InvalidDepositoryShares,
-    #[msg("The provided mercurial vault does not match the Depository's one.")]
+    #[msg("The Profits beneficiary collateral provided does not match the depository's one.")]
+    InvalidProfitsBeneficiaryCollateral,
```

Comment:
nit: this diff should disappear when rebasing with master

Reply:
same